develop-docs/sdk/expected-features/mcp-instrumentation/index.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: MCP Instrumentation
+---
+
+The MCP Server module instruments Anthropic's Model Context Protocol (MCP) SDKs. At the moment it only supports the [MCP Typescript SDK](https://github.com/modelcontextprotocol/typescript-sdk/).
+
+## Features
+
+- [Tracing](./tracing)
+- [Errors](./errors)
+
+## Current limitations
+
+Currently, it doesn't support automatic instrumentation due to [this issue](https://github.com/getsentry/sentry-javascript/issues/16052) related to `import-in-the-middle`.
```

Comment:
this is javascript specific, I'd avoid bringing this up here, as these docs are meant to apply to all sdk implementations (current and future).